content/docs/command-reference/exp/init.md

```diff
@@ -64,71 +80,123 @@ $ dvc exp init './another_script.sh $MYENVVAR'
+  default value (`metrics.json`).
 
-  `dl` stage is intended for use in deep-learning scenarios, where metrics and
-  plots are tracked by [dvclive](/doc/dvclive) and supports tracking progress
-  while training a deep-learning model with
-  [checkpoints](/doc/command-reference/exp/run#checkpoints).
+- `--plots` - set the path to the file or directory where the plots produced by
+  your experiment can be found (if any). Overrides other configuration and
+  default value (`plots/`).
 
-- `-n <stage>`, `--name <stage>` - specify a custom name for the stage generated
-  by this command (e.g. `-n train`). The default is `train`.
+- `--live` - configure the `path` directory for [DVCLive](/doc/dvclive). This is
+  where experiment logs will be written. Overrides other configuration and
+  default value (`dvclive/`).
 
-  Note that the stage name can only contain letters, numbers, dash `-` and
-  underscore `_`.
+  > This only has an effect when used with `--type=dl`.
 
-- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
-  asking for confirmation.
+- `--explicit` - do not assume default locations of project dependencies and
+  outputs. You'll have to provide specific locations via other options or
+  `dvc config exp`. In `--interactive` this removes default values from prompts.
 
-- `--run` - runs the experiment after initializing it.
+- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
+  asking for confirmation (same as `dvc stage add --force`).
 
 - `-h`, `--help` - prints the usage/help message, and exit.
 
 - `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example: interactive mode
+
+Let's prepare a raw data ingestion script to start running experiments on it.
+
+```dvc
+$ dvc exp init --interactive --name ingest
+This command will guide you to set up a train stage in dvc.yaml.
+See https://s.dvc.org/g/pipeline-files.
+
+Command to execute: python src/ingest.py
+
+Enter the paths for dependencies and outputs of the command.
+DVC assumes the following workspace structure:
+├── data
+├── metrics.json
+├── models
+├── params.yaml
+├── plots
+└── src
+
+Path to a code file/directory [src, n to omit]: src/in.py
+Path to a data file/directory [data, n to omit]: data/raw
+Path to a model file/directory [models, n to omit]: n
+Path to a parameters file [params.yaml, n to omit]:
+Path to a metrics file [metrics.json, n to omit]: n
+Path to a plots file/directory [plots, n to omit]: n
+...
+```
+
+Notice that the code and data locations were specified above, to avoid using the
+defaults which are too broad. `params.yaml` exists in the workspace and is used
+by our code in question, so the default is used. Models, metrics, and plots are
+omitted as there are none for an ingestion process.
+
+The resulting `dvc.yaml` file is simply:
```

Comment:
Might be worth mentioning pipelines briefly to explain what's in `dvc.yaml`.

Reply:
I'm actively avoiding mixing in that concept here but it is mentioned strategically once in the Description. I did elaborate on this sentence a bit though.